nwb_conversion_tools/utils/basenwbephyswriter.py

```diff
@@ -110,6 +114,40 @@ def _get_unit_waveforms_templates(self, unit_id, mode="mean"):
         """
         pass
 
+    def get_default_nwbfile_metadata(self):
+        """
+        Returns structure with defaulted metadata values required for a NWBFile.
+        """
+        metadata = dict(
+            NWBFile=dict(
+                session_description="no description",
+                session_start_time=datetime(1970, 1, 1),
+                identifier=str(uuid.uuid4()),
+            ),
+            Ecephys=dict(
+                Device=[dict(name="Device", description="Ecephys probe. Automatically generated.")],
+                Electrodes=[],
+                ElectrodeGroup=[],
+                ElectricalSeries_raw=dict(
+                    name="ElectricalSeries_raw",
+                    description="Raw acquired data",
+                    comments="Generated from SpikeInterface::NwbRecordingExtractor",
+                ),
+                ElectricalSeries_processed=dict(
+                    name="ElectricalSeries_processed",
+                    description="Processed data",
+                    comments="Generated from SpikeInterface::NwbRecordingExtractor",
+                ),
+                ElectricalSeries_lfp=dict(
+                    name="ElectricalSeries_lfp",
+                    description="Processed data - LFP",
+                    comments="Generated from SpikeInterface::NwbRecordingExtractor",
+                ),
```

Comment:
I don't think these fields should be present in the metadata unless there is processed or lfp data